### DISCUSSION TITLE
Removing the Close Button 'X' from the Modal

### DISCUSSION THREAD

**Discussion by roy-wong-d24h** [picocss/pico#507] [Help]
Hello, I would like to remove the close button 'X' from the modal and ensure users cannot close the modal by pressing the escape key. Any suggestions on how to achieve this?

**Comment by CrlH** [comment]
As per the [docs](https://picocss.com/docs/modal), omitting the `<button rel="prev">` in your `<header>` will remove the "x" button on top.

Note that closing with `Esc` is default behavior of  `<dialog>`, you can read out it on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog). If you'd like to prevent it, I'd recommend the suggestion of docs to take control over the element using JavaScript.
